### ISSUE TITLE
finetune interuption : paddle::pybind::ThrowExceptionToPython(std::__exception_ptr::exception_ptr)

### ISSUE THREAD

**Issue by hoahoa1808** [PaddlePaddle/PaddleOCR#10011]
I'm finetune text recognition model with (CAN) on dataset im2latex100k.
I set up following [the tutorial](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/algorithm_rec_can_en.md), with costom config as below:
```Global:
  use_gpu: True
  epoch_num: 240
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ../output/can/
  save_epoch_step: 1
  # evaluation is run every 1105 iterations (1 epoch)(batch_size = 8)
  eval_batch_step: [0, 1238]
  cal_metric_during_train: True
  pretrained_model:
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/datasets/crohme_demo/hme_00.jpg
  # for data or label process
  character_dict_path: ../data/data_rec/latexdict.txt
  max_text_length: 50          #??????????
  infer_mode: False
  use_space_char: False
  save_res_path: ../output/predicts_can.txt


Architecture:
  model_type: rec
  algorithm: CAN
  in_channels: 1
  Transform:
  Backbone:
    name: DenseNet 
    growthRate: 24
    reduction: 0.5
    bottleneck: True
    use_dropout: True
    input_channel: 1 
  Head:
    name: CANHead
    in_channel: 684
    out_channel: 579             #INDEX
    max_text_length: 50          #??????
    ratio: 16
    attdecoder:
      is_train: True
      input_size: 256
      hidden_size: 256
      encoder_out_channel: 684
      dropout: True
      dropout_ratio: 0.5
      word_num: 579             #DIC
      counting_decoder_out_channel: 579 ##DIC
      attention:
        attention_dim: 512
        word_conv_kernel: 1
   
Loss:
  name: CANLoss

PostProcess:
  name: CANLabelDecode

Metric:
  name: CANMetric
  main_indicator: exp_rate

Optimizer:
  name: Momentum
  momentum: 0.9
  clip_norm_global: 100.0
  lr:
    name: TwoStepCosine
    learning_rate: 0.001
    warmup_epoch: 1
  weight_decay: 0.0001

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ../data/
    label_file_list: ["../data/data_rec/train.txt"]
    transforms:
      - DecodeImage:
          channel_first: False
      - NormalizeImage:
          mean: [0,0,0]
          std: [1,1,1]
          order: 'hwc'
      - GrayImageChannelFormat: 
          inverse: True
      - CANLabelEncode:
          lower: False
      - KeepKeys:
          keep_keys: ['image', 'label']
  loader:
    shuffle: True
    batch_size_per_card: 4
    drop_last: False
    num_workers: 4
    collate_fn: DyMaskCollator

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ../data/
    label_file_list: ["../data/data_rec/test.txt"]
    transforms: 
      - DecodeImage:
          channel_first: False
      - NormalizeImage:
          mean: [0,0,0]
          std: [1,1,1]
          order: 'hwc'
      - GrayImageChannelFormat:
          inverse: True
      - CANLabelEncode:
          lower: False
      - KeepKeys:
          keep_keys: ['image', 'label']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1
    num_workers: 4
    collate_fn: DyMaskCollator
```
after 80 inters, I got an error as:
```
 paddle::pybind::ThrowExceptionToPython(std::__exception_ptr::exception_ptr)
```
without any information.
is there anyone can help me? 

- 系统环境/System Environment：Ubuntu, Anaconda3, GPU
- GPU Tesla T4 16Gb, Cuda Version: 11.7, Runtime API Version: 10.2, cuDNN Version: 7.6.
- 版本号/Version：Paddle：paddlepaddle-gpu      2.4.2,  PaddleOCR： install from source

**Comment by nissansz** [comment]
what is the accuracy difference between densenet and resnet34?
